template/default.tmpl

```diff
@@ -65,8 +65,8 @@ AlertmanagerUrl:
 {{ define "wechat.default.to_party" }}{{ end }}
 {{ define "wechat.default.to_tag" }}{{ end }}
 {{ define "wechat.default.agent_id" }}{{ end }}
-
-
+{{ define "wechat.default.agent_id" }}text{{ end }}
```

Comment:
this line is a duplicate of the previous one and you're missing a few others fields.

Reply:
fixed